src/js/select2/a11y/a11ySingle.js

```diff
@@ -0,0 +1,19 @@
+define([
+  '../utils'
+], function (Utils) {
+  function A11ySingle (decorated, $element, options) {
+    decorated.call(this, $element, options);
+  }
+
+  A11ySingle.prototype.bind = function (decorated, data) {
+
+    // If orginal select had aria-describedby, add to select2-selection
+    if (this.$element.attr('aria-describedby') != null) {
```

Comment:
Although this is pretty risk-free, I think it's worth getting into the habit of strict comparison and change this to `!== null`